Examples/Profiles/Profile-example.fsh

```diff
@@ -1,4 +1,39 @@
 // @Name: Profile Example
+// @Description: Example of a profile of the Patient resource. This example includes a few of the most commonly used constraints and documentation features of FHIR profiles.
 
-// Nothing yet! 
-// Help wanted -- your example can go here. 
\ No newline at end of file
+Profile:     ExamplePatientProfile
+Id:          example-patient-profile
+Parent:      Patient
+Title:       "Example Patient Profile"
+Description: "Example of a profile of Patient"
+
+// The contents of `^short` appears in the \"Description & Constraints\" column in the differential or snapshot tables of a built Implementation Guide.
+// In this case, `. ^short` sets this for the root level of the element hierarchy displayed in the table. `someElement ^short` will set this text for a specific
+// element, as is seen below.
+* . ^short = "A short description of the profile"
```

Comment:
Suggest setting short and definition on a non-root element. That's by far the most common use case.

Reply:
Moved this down below setting `^short` and `^definition` on `name` so the more common usage comes first.

I like leaving this in as it's non-obvious that one can do this on the root element, and it's easily ignorable at the end now.

Reply:
I'm fine with leaving the demonstration of setting root in, but I'd leave it at the top myself.  The root is at the top when you view the JSON, XML, and snapshot/differential pages in the IG.  It feels weird to me to have the root element listed below the other elements (that are technically its children).